base/src/org/compiere/wf/MWFActivity.java

```diff
@@ -1213,30 +1212,23 @@ else if (!(doc.processIt (DocAction.ACTION_Approve)))
 			}
 			// Send Approval Notification
 			if (newState.equals(StateEngine.STATE_Aborted)) {
-				MUser to = new MUser(getCtx(), doc.getDoc_User_ID(), null);
-				
-				// send email
-				if (to.isNotificationEMail()) {
-					MClient client = MClient.get(getCtx(), doc.getAD_Client_ID());
-					client.sendEMail(doc.getDoc_User_ID(), Msg.getMsg(getCtx(), "NotApproved")
-							+ ": " + doc.getDocumentNo(), 
-							(doc.getSummary() != null ? doc.getSummary() + Env.NL : "" )
+				//	Get instance for notifier
+				DefaultNotifier notifier = (DefaultNotifier) QueueLoader.getInstance().getQueueManager(DefaultNotifier.QUEUETYPE_DefaultNotifier)
+						.withContext(Env.getCtx())
+						.withTransactionName(trx.getTrxName());
+				//	Send notification to queue
+				notifier
+					.clearMessage()
+					.withApplicationType(DefaultNotifier.DefaultNotificationType_UserDefined)
```

Comment:
Hi @yamelsenih, in this case, when you use the "withApplicationTYpe" with the option "DefaultNotificationType_UserDefined", you don't have the user stablished.

Reply:
Hi @yamelsenih 
When you do not specify an email address (Text in Workflow Node) you get an error.

Reply:
Hi @all the user defined just allows define directly from user notification type and a notification can be send by email, telegram and other media type depending of user defined and supported client